sdk/core/core-http/src/operationOptions.ts

```diff
@@ -67,7 +67,6 @@ export function operationOptionsToRequestOptionsBase<T extends OperationOptions>
   }
 
   if (tracingOptions) {
-    result.spanOptions = tracingOptions.spanOptions;
```

Comment:
I want to call out that there will be a tracingPolicy change here, we would need to fetch the `az.namespace` attribute from context instead of spanOptions but I haven't gotten there yet, still TODO